ci/release/update-version.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+########################
+# .gitmodules branch Updater #
+########################
+
+## Usage
+# bash update-version.sh <new_version>
+
+
+# Format is YY.MM.PP - no leading 'v' or trailing 'a'
+NEXT_FULL_TAG=$1
+
+# Get current version
+CURRENT_TAG=$(git tag --merged HEAD | grep -xE '^v.*' | sort --version-sort | tail -n 1 | tr -d 'v')
+CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
+CURRENT_MINOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[2]}')
+CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}')
+CURRENT_SHORT_TAG=${CURRENT_MAJOR}.${CURRENT_MINOR}
+
+#Get <major>.<minor> for next version
+NEXT_MAJOR=$(echo $NEXT_FULL_TAG | awk '{split($0, a, "."); print a[1]}')
+NEXT_MINOR=$(echo $NEXT_FULL_TAG | awk '{split($0, a, "."); print a[2]}')
+NEXT_SHORT_TAG=${NEXT_MAJOR}.${NEXT_MINOR}
+
+echo "Preparing release $CURRENT_TAG => $NEXT_FULL_TAG"
+
+# Inplace sed replace; workaround for Linux and Mac
+function sed_runner() {
+    sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
+}
+
+# .gitmodules update
+sed_runner "s/= branch.*/= branch-${NEXT_FULL_TAG}/g" ../../.gitmodules
```

Comment:
```suggestion
sed_runner "s/= branch.*/= branch-${NEXT_SHORT_TAG}/g" .gitmodules
```

Two quick changes, otherwise looks good:

1) The script is usually run from the project's root directory, so we need to change the path to be relative to the root ([similar to cudf](https://github.com/rapidsai/cudf/blob/d68e626dff00283431ac6ac5fd799a3a5ed8dbbe/ci/release/update-version.sh#L61))
2) We want this to be updated to the `NEXT_SHORT_TAG`, which is equivalent to `${NEXT_MAJOR}.${NEXT_MINOR}`